modules/material-management/test/cryptographic_material.test.ts

```diff
@@ -97,7 +77,6 @@ describe('decorateCryptographicMaterial', () => {
     test.setUnencryptedDataKey(dataKey, {
       keyNamespace: 'k',
       keyName: 'k',
-      flags: KeyringTraceFlag.WRAPPING_KEY_GENERATED_DATA_KEY,
     })
```

Comment:
Same as above: shouldn't this keyring trace (second argument) be deleted entirely?